Info.pm

```diff
@@ -652,7 +652,8 @@ sub _get_v1tag {
 				# Remove garbage and retry
 				# (string is truncated in the
 				# middle of a multibyte char?)
-				$value =~ s/(.)$//;
+				#$value =~ s/(.)$//; # error https://rt.cpan.org/Ticket/Display.html?id=101620
```

Comment:
It's right below, line 656 
![screenshot from 2017-06-11 09 28 40](https://user-images.githubusercontent.com/500/27009192-6b0fe6d4-4e88-11e7-8158-d786b0158ece.png)